.github/workflows/main.yml

```diff
@@ -37,6 +37,13 @@ jobs:
         run: bin/rails db:schema:load
       - name: Run tests
         run: bin/rake test
+      - name: Push test coverage to Code Climate
+        uses: paambaati/codeclimate-action@v6.0.0
+        env:
+          CC_TEST_REPORTER_ID: ${{ secrets.CC_TEST_REPORTER_ID }}
+        with:
+          coverageCommand: bin/rake test
```

Comment:
Is this doing the same as the first command, or is it JUST for the coverage page?: `run: bin/rake test`

Reply:
Great catch that was actually redundant, it was running the tests all over again then pushing the report.

I've combined both